sysid-application/src/main/native/cpp/telemetry/TelemetryManager.cpp

```diff
@@ -129,6 +133,12 @@ void TelemetryManager::EndTest() {
                    m_settings.unitsPerRotation;
         stream << "The encoder reported traveling " << p << " " << units << ".";
       }
+
+      if (m_params.overflow) {
+        stream << "NOTE: the robot stopped recording data early because the "
```

Comment:
Looks like there should be a linebreak before this note. Also, the contents should be placed in a wrapped textbox so it doesn't go past the end of the box as in the screenshot.